packages/utils/src/media/CanvasRenderTarget.ts

```diff
@@ -69,27 +75,43 @@ export class CanvasRenderTarget
      * The width of the canvas buffer in pixels.
      * @member {number}
      */
-    get width(): number
+    get width(): number | null
```

Comment:
`get width()` returning `null` can be confusing. Maybe we should check if `this.canvas === null` (this means the `CanvasRenderTarget` has been destroyed) and throw a `TypeError`.